### ISSUE TITLE
Question: Can you edit the html/css/js to the embed player?

### ISSUE THREAD

**Issue by rashawntemple** [AzuraCast/AzuraCast#695]
Can you edit the html/css/js to the embed player? If so, what the name to the those files and location?
![radio](https://user-images.githubusercontent.com/42072386/43654363-fb201b2a-9718-11e8-8864-1b1ce9727697.jpg) 

My goal is to put more focus in the presentation of the album art.

**Comment by SlvrEagle23** [comment]
@rashawntemple The "Custom Branding" page allows you to specify both custom CSS and custom JavaScript to be included on all public-facing pages, including the embed pages. The `<body>` tag on the minimal iframe-style embeds has the class `embed`, so you could just target those pages specifically with:

```css
body.embed {
}
```

in the custom CSS.

**Comment by pwrj** [comment]
So, what code do use to change the text color of now playing? Asking for noob. lol

**Comment by pwrj** [comment]
> @rashawntemple The "Custom Branding" page allows you to specify both custom CSS and custom JavaScript to be included on all public-facing pages, including the embed pages. The `<body>` tag on the minimal iframe-style embeds has the class `embed`, so you could just target those pages specifically with:
> 
> ```css
> body.embed {
> }
> ```
> 
> in the custom CSS.

So, what code do use to change the text color of now playing? Asking for noob. lol

**Comment by Vaalyn** [comment]
`body.embed .now-playing-title` and `body.embed .now-playing-artist`

**Comment by pwrj** [comment]
Thanks, you're a life saver!!

**Comment by pwrj** [comment]
> `body.embed .now-playing-title` and `body.embed .now-playing-artist`

I mean what code do I put in the Custom CSS for Public Pages. Say, I want the text color to be white.

**Comment by SlvrEagle23** [comment]
@pwrj The custom CSS field supports standard CSS, and there are countless places online you can go to learn basic CSS markup.

Given your example, the code could look like:

```css
body.embed .now-playing-title,
body.embed .now-playing-artist {
   color: #FFFFFF;
}
```

**Comment by pwrj** [comment]
> @pwrj The custom CSS field supports standard CSS, and there are countless places online you can go to learn basic CSS markup.
> 
> Given your example, the code could look like:
> 
> ```css
> body.embed .now-playing-title,
> body.embed .now-playing-artist {
>    color: #FFFFFF;
> }
> ```

That's exactly what I needed. Thanks a million!

**Comment by radiorueda** [comment]
> Can you edit the html/css/js to the embed player? If so, what the name to the those files and location?
> ![radio](https://user-images.githubusercontent.com/42072386/43654363-fb201b2a-9718-11e8-8864-1b1ce9727697.jpg)
> 
> My goal is to put more focus in the presentation of the album art.

hi @rashawntemple 
this is my first post in github, i am not a code guy, but i am trying to setup a nice player for our www.radiorueda.com
i would like to know if you solved somehow this issue. we would like to have also more presence of the cover art. thank you!

**Comment by FringeRadioNetwork** [comment]
> hi @rashawntemple
> this is my first post in github, i am not a code guy, but i am trying to setup a nice player for our [www.radiorueda.com](http://www.radiorueda.com)
> i would like to know if you solved somehow this issue. we would like to have also more presence of the cover art. 
You mean the background image on the public page?  I love the Unicorn!  It's really bad luck to kill a unicorn... I wanted a heavy brand though.  I did not use css to do that though.  I installed a remote GUI and replaced hexbg.png found in this filepath:  /var/lib/docker/volumes/azuracast_www_data/_data/web/static/img/  and in the command line manual install ( which I LOVE! )  /var/azuracast/www/web/static/img/  But I plan on setting up another unicorn based domain and keeping most holy unicorn on there because I love it.  Shout out to Tyson Tan.  Great artist!
